pyServer/manifests/windows/sql-iaas

```diff
@@ -17,6 +17,8 @@ copy,/WindowsAzure/Logs/Plugins/Microsoft.SqlServer.Management.SqlIaaSAgent/*/Co
 copy,/WindowsAzure/Logs/Plugins/Microsoft.SqlServer.Management.SqlIaaSAgent/*/ExtensionLog*.log
 copy,/WindowsAzure/Logs/Plugins/Microsoft.SqlServer.Management.SqlIaaSAgent/*/SqlCustomerSupportLogs/*
 
+copy,/SQLIaaSExtension/Log/*
```

Comment:
How big this can be in size?
As it has "*" ho many different files are you expecting? And is there any chances that those log files can contains any type of customer secrets?
If are you sure that these logs can't have any secrets then we can add "noscan" at the end of 'copy' line to avoid credscan for that file.
Also you need to generate this PR from the branch created form master not from forked repo. Please follow below steps.
https://github.com/Azure/azure-diskinspect-service#how-to-contribute